content/tagging/_index.md

```diff
@@ -38,49 +38,44 @@ A few best practices on tags:
+    
+    | Tag                      | Key                | Value            |
+    |--------------------------|--------------------|------------------|
+    | `service:database:mysql` | `service`          | `database:mysql` |
+    | `service_database:mysql` | `service_database` | `mysql`          |
 
-    * `role:database:mysql` is parsed as **key**:`role` , **value**:`database:mysql`
-    * `role_database:mysql` is parsed as **key**:`role_database` , **value**:`mysql`
-
-    Examples of commonly used metric tag keys are `env`, `instance`, `name`, and `role`.
-
-5. `device`, `host`, and `source` are **reserved tag keys** and cannot be specified in the standard way.
+5.  **Reserved tag keys** `device`, `host`, and `source` cannot be used in the standard way.
 
 6. Tags shouldn't originate from unbounded sources, such as EPOCH timestamps or user IDs. These tags may impact platform performance and billing.
 
-## Applying Tags
-
-Tags may be added using any (or all) of the following methods:
-
-* Agent Tags (`datadog.yaml`)
-* [DogStatsD][2] Tags
-* Integration/Check Tags (each check on the local host supports tags by editing the yaml)
-* Tags generated by other services such as [AWS][3], [Azure][4], [GCE][5], etc.
-* Tags in the [API][6] - note other endpoints support tags as well such as Events and Metrics
-* [Chef Roles][7] and [Puppet][8] Tags (Chef and Puppet use the API - this may obviously be extended to other configuration management tools by you or Datadog)
-* Manually adding tags using the [Infrastructure List][9] (hover over host->select "Inspect"->"Edit Tags")
-
-## Examples
-
-Here is an example of tags using the time-series chart editor. For the first screenshot, no tags have been applied, and we're observing average CPU across all hosts:
-
-{{< img src="tagging/Tags_1.png" alt="Tags_1" responsive="true" style="width:75%;">}}
+## Assigning Tags
 
-In this next example, we've applied a tag (`region:eastus`) that enables Datadog to look at CPU across the US East Region. We've used region as an example, but you could use any arbitrary tag, including application, service, environment, etc.
+Tags may be [assigned][21] using any (or all) of the following methods:
 
-{{< img src="tagging/Tags_2.png" alt="Tags_2" responsive="true" style="width:75%;">}}
+* [Configuration Files][22]
+* [Environment Variables][23]
+* [UI][24]
+* [API][25]
+* [DogStatsD][26]
+* [Integration Inheritance][27]
 
-In this last example, we've used the second empty field labeled as "everything" by option to show an individual timeseries line for each host. Now we're seeing server CPU for individual hosts running in the US East Region.
+## Using Tags
 
-{{< img src="tagging/Tags_3.png" alt="Tags_3" responsive="true" style="width:75%;">}}
+After you have [assigned tags][21] at the host and [integration][11] level, you can start using them to filter and group in interesting ways. There are several places you can use tags. Explore the various areas:
```

Comment:
We should avoid using `you can` as much as possible:
r/`level, you can start using them`/ `level, start using them`

`in interesting ways` is too vague => `to filter and group your metrics, traces, and logs.`

`There are several places you can use tags. Explore the various areas:` those two sentences are redundant, => `Tags are used in the following areas of your Datadog platform, refer to the dedicated [Using Tags documentation](/tagging/using_tags) to learn more:`

Reply:
btw, idk if we could also use an array here to give a quick snippet of tags usage in those different areas.